### PR TITLE
feat: update `tailwindcss` spec to latest version

### PR DESCRIPTION
**Pull request by fedeci**

### CODE REVIEW

File: dev/tailwindcss.ts [hunk 1]
```diff
@@ -32,16 +32,22 @@ const buildOptions: Fig.Option[] = [
...
   {
     name: "--purge",
+    description:
+      "Content paths to use for removing unused classes. [Deprecated]: use `--content` instead",
+  },
+  {
+    name: "--purge",
+    description:
+      "Content paths to use for removing unused classes. [Deprecated]: use `--content` instead",
+  },
```
Comment (QuiiBz) [inline dev/tailwindcss.ts:42]: Looks like this option is duplicated

**Review by QuiiBz — CHANGES_REQUESTED** [review]

File: dev/tailwindcss.ts [hunk 2]
```diff
@@ -32,16 +32,22 @@ const buildOptions: Fig.Option[] = [
   },
   {
     name: "--purge",
+    description:
+      "Content paths to use for removing unused classes. [Deprecated]: use `--content` instead",
+  },
+  {
+    name: "--purge",
+    description:
+      "Content paths to use for removing unused classes. [Deprecated]: use `--content` instead",
```
Comment (fedeci) [inline dev/tailwindcss.ts:41]:
```suggestion
    name: "--content",
    description: "Content paths to use for removing unused classes instead",
```

File: dev/tailwindcss.ts (hunk 1 above)
Comment (fedeci) [reply dev/tailwindcss.ts]: 🤦‍♂️

File: dev/tailwindcss.ts [hunk 3]
```diff
@@ -32,16 +32,21 @@ const buildOptions: Fig.Option[] = [
...
+  },
+  {
+    name: "--content",
+    description: "Content paths to use for removing unused classes instead",
+  },
+  {
+    name: "--content",
     description: "Content paths to use for removing unused classes",
   },
```
Comment (QuiiBz) [inline dev/tailwindcss.ts:45]: Looks like there are still two options that are the same 😅

**Review by QuiiBz — CHANGES_REQUESTED** [review]

File: dev/tailwindcss.ts (hunk 3 above)
Comment (fedeci) [reply dev/tailwindcss.ts:45]: Damn😆

File: dev/tailwindcss.ts [hunk 4]
```diff
@@ -32,16 +32,21 @@ const buildOptions: Fig.Option[] = [
...
   {
     name: "--purge",
+    description:
+      "Content paths to use for removing unused classes. [Deprecated]: use `--content` instead",
+  },
+  {
+    name: "--content",
+    description: "Content paths to use for removing unused classes instead",
+  },
```
Comment (fedeci) [inline dev/tailwindcss.ts:41]:
```suggestion
```

File: dev/tailwindcss.ts (hunk 3 above)
Comment (fedeci) [reply dev/tailwindcss.ts]: Fixed

**Review by QuiiBz — APPROVED** [review]
LGTM 🚀

**Pull request merged**